chore: create layout component (WEB-40)

climaxmba opened the pull request:
# chore: create layout component (WEB-40): #86 
> ## ✅ _Ready for review_

climaxmba commented on src/app/components/layout/layout.tsx:
```diff
@@ -0,0 +1,17 @@
+
+export default function Layout() {
+  return (
+    <>
+      <Navbar
+        label={{
+          lblHome: 'Dallas Software Developers',
+          lblCommunity: 'Community Impact',
+          lblContact: 'Contact Us',
+          lblMeetup: 'Meetups',
+        }}
+      />
+      <main></main>
+    </>
```
Good day @GetBlackBoxSolutions! Please I need to know if this component should return a `<div>` as the root element or is it alright this: `<>{ children }</>`

GetBlackBoxSolutions reviewed (changes requested):
This is very close. Please look at the NextJs documentation. 

https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern

climaxmba commented:
> This is very close. Please look at the NextJs documentation.
> 
> https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern

I have added the `children` prop and nested `children` in `<main>`. By the way, would it be ok if I added a test for the component?
